pkg/controllers/gitrepository/controller.go

```diff
@@ -13,7 +15,7 @@ import (
 	"github.com/cnoe-io/idpbuilder/pkg/util"
 	"github.com/go-git/go-git/v5"
 	"github.com/go-git/go-git/v5/plumbing/object"
-	"github.com/go-git/go-git/v5/plumbing/transport/http"
+	gohttp "github.com/go-git/go-git/v5/plumbing/transport/http"
```

Comment:
Why name it `gohttp`? Should be something like `gogithttp` if you want to be explicit.

Reply:
fixed.